src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperServiceImpl.java

```diff
@@ -142,12 +142,16 @@ public void restartFloodlight(String region) {
 
     @Override
     public List<FloodlightResourceAddress> knockoutSwitch(Switch sw, List<String> regions) {
+        throw new RuntimeException("This method is not allowed to run on hardware environment. " +
```

Comment:
As an option you can throw `UnsupportedOperationException`. From the description I understood that it's ok to use it on some other environments, should there be some if-condition?

Reply:
If I get the code right, dependency injections replaces if's in this case.
Changed the type of exception thrown.